src/ConfigurationContext.tsx

```diff
@@ -2,13 +2,16 @@ import React, { FunctionComponent, createContext, useContext } from "react"
 import { Platform } from "react-native"
 import env from "react-native-config"
 
+const USA_EMERGENCY_NUMBER = "911"
```

Comment:
What are your thoughts on not providing a default?

We would need to handle the null cases by not rendering the emergency call buttons, but perhaps that's better behavior than defaulting to 911?

Reply:
No thoughts other than editing 33 files to add the value 😬  but I agree it breaks with the pattern we have.

Reply:
100%, lets not edit the 33 files right now, but lets cut a card to consider for the a future update.

Reply:
No worries, I updated the PR with that consideration and added the values on the resources 👍